api/api_test.go

```diff
@@ -530,6 +535,67 @@ func blockedHidden(name string, blocked, hidden bool) tasktemplate.TaskTemplate
 	}
 }
 
+func Test_staticMiddleware(t *testing.T) {
+	ctx := context.Background()
+
+	ginEngine := gin.Default()
+	ginEngine.
+		Group("/", api.StaticFilePatternReplaceMiddleware("static.go", "___test_suite___")).
+		StaticFS("/", http.Dir("./"))
+
+	srv := &http.Server{Addr: ":12129", Handler: ginEngine}
```

Comment:
Why ? Can't you do
```
tester := iffy.NewTester(t, ginEngine)
```

You should not have to start a real http Server for testing (and not on an hardcoded TCP port...)

Reply:
Wasn't sure I could do this, I prefer, thanks for the tip. Fixed